### PR TITLE
Refactor project card/data card to be more useful

### PR DESCRIPTION
**Pull request by alanmoo**

### CODE REVIEW

File: app/react/components/data-card/data-card.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  getDefaultProps() {
+    return{
+      showPicture: false
+    };
+  },
+  render() {
+
+    return (
+      <div className={`data-card-wrapper ` + this.props.className}>
```
Comment (gvn) [inline app/react/components/data-card/data-card.jsx]: You can just leverage your template string instead of doing concatenation.

Comment (alanmoo) [inline app/react/components/data-card/data-card.jsx]: lol oops

File: app/react/components/data-card/data-card.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  display: flex;
+}
+
+.data-card {
+  background: $white;
+  position: relative;
+  margin: 2rem 0;
+  padding: 1rem;
+  font-weight: 300;
+  width: 100%;
```
Comment (gvn) [inline app/react/components/data-card/data-card.scss]: Is this needed if it's already display:block?

**Comment by gvn** [comment]
LGTM

**Pull request merged**